.github/workflows/node.js.yml

```diff
@@ -27,3 +27,25 @@ jobs:
         npm run build
         npm run fmt:check
         npm run test:ci
+    - id: coverage
+      run: |
+        value=$(npm run test:ci | grep -oE '^│\s*total\s+\│\s*([0-9.]+)%' | grep -oE '[0-9]+(\.[0-9]+)?%' | cut -d'.' -f1)
+        echo "coverage=$value" >> $GITHUB_OUTPUT
+
+    - name: Update README
+      run: |
+        "${GITHUB_WORKSPACE}/.github/coverage.sh"
+
+    - name: Commit changes
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: "Generate coverage badge"
+        file_pattern: "README.md"
+        # Commit code using massabot account that can bypass push and MR restriction
+        commit_author: massabot <info@massa.net>
+ 
+
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do youneed github token ?

Reply:
To push with the github-actionbot from the script